2996_reflection/reflection.md

```diff
@@ -32,10 +32,22 @@ Since [@P2996R8]:
 * specifically state that `define_aggregate` cannot be used for a class currently being defined
 * `members_of($closure-type$)` returns an unspecified sequence of reflections
 * introduced a strong ordering on evaluations during constant evaluation
-* core wording improvements
-  * classify the injection of a declaration as a side effect
-  * rework [lex.phases]: replace "semantically follows" notion; define when a plainly constant-evaluated expression must be evaluated in terms of side effects and reachability. Inline "semantically sequenced" into [expr.const] conditions that make an injected declaration ill-formed.
+* assertions of `$static_assert-declaration$`s are not plainly constant-evaluated
+* core wording changes
```

Comment:
I feel like this is just... way, way too detailed a revision history.

Reply:
I cut it down to only cite more substantial changes, but 1) we got feedback from Core that the revision history was woefully incomplete, and 2) R8 was a biiiiiig revision for wording.

Reply:
Really? My level of revision history is always like "wording" and nobody has ever complained about it. I guess this paper is bigger, but still. At some point we should set things up so that people can just look at the diff if they want to...